packages/experimental/Avatar/src/JSAvatar.tsx

```diff
@@ -54,7 +55,7 @@ export const JSAvatar = compose<JSAvatarType>({
           )}
           {showRing && !transparentRing && <Slots.ring />}
           {activeAppearance === 'glow' && <Slots.glow />}
-          {!!iconSource && !!iconSource.uri && <Slots.icon source={iconSource} />}
+          <PresenceBadge size="small" shape="circular" presence={presence} isOutOfOffice={isOutOfOffice} />
```

Comment:
Hmm I'm still curious about why this can't be a slot and why the framework complains about it, but I'll look into that on the side and I don't think it needs to block this PR.

Reply:
It can. There's an error when I use it as a slot. I'll try to fix it today